frontend/src/component/project/ProjectList/ProjectList.tsx

```diff
@@ -172,6 +205,36 @@ export const ProjectListNew = () => {
             header={
                 <PageHeader
                     title={`Projects (${projectCount})`}
+                    leftActions={
+                        <ConditionallyRender
```

Comment:
This looks like it could be a separate component.

Reply:
I'm not entirely sure I understand. What do you mean?